internal/tests/notrequireargsinsubcommands_test.go

```diff
@@ -0,0 +1,36 @@
+package tests
+
+import (
+	"context"
+	"testing"
+
+	"github.com/conductorone/baton-sdk/pkg/field"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCallSubCommand(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), timeoutIn)
+	defer cancel()
+
+	requiredField := field.StringField("name", field.WithRequired(true))
+	carrier := field.NewConfiguration(
+		[]field.SchemaField{
+			requiredField,
+		},
+	)
+
+	t.Run("should run «help» sub-command successfully", func(t *testing.T) {
+		_, err := entrypoint(ctx, carrier, "help")
+		require.NoError(t, err)
+	})
+
+	t.Run("should run «capabilities» sub-command without success", func(t *testing.T) {
```

Comment:
Can you add a test where you call `entrypoint()` with `"capabilities"` and then _not_ error?
Something like:
```go
t.Run("should run «capabilities» sub-command without success", func(t *testing.T) {
  _, err := entrypoint(ctx, carrier, "capabilities", "name")
  require.NoError(t, err)
})
```

Reply:
I can, and I should!